packages/unlock-js/src/v7/getLock.js

```diff
@@ -79,5 +85,22 @@ export default async function (address, provider) {
   }
   delete update.tokenAddress
 
+  // check if compromised (Nov 22nd, 2021 attack)
+  if (update.unlockProtocol) {
+    const compromisedContracts = {
+      100: '0x3C6e461341AdbF7C0947085e86B4A6f35Ff2F801',
```

Comment:
I think this is not the right address. The previous one was also `0x14bb3586ce2946e71b95fe00fc73dd30ed830863` on xdai,